web/src/components/DLP/DandisetOwnersDialog.vue

```diff
@@ -141,6 +141,41 @@
         </div>
       </v-col>
     </v-row>
+    <v-dialog
+      v-model="adminWarningDisplay"
+      max-width="60vw"
+      persistent
+    >
+      <v-card class="pb-3">
+        <v-card-title class="text-h5 font-weight-light">
+          WARNING
+        </v-card-title>
+        <v-divider class="my-3" />
+        <v-card-text>
+          This action will modify the owners of this dandiset. You are not an owner of this
+          dandiset, but as an admin you may still perform this action.
+        </v-card-text>
+        <v-card-text>
+          Would you like to proceed?
+        </v-card-text>
+
+        <v-card-actions>
+          <v-btn
+            color="error"
+            depressed
+            @click="save()"
```

Comment:
Usually you just pass the function itself
```suggestion
            @click="save"
```